web/template/supporter/organisation_details.gohtml

```diff
@@ -20,16 +20,73 @@
 
       <h1 class="govuk-heading-xl">{{ tr .App "manageOrganisation" }}</h1>
 
-      <div class="govuk-tabs app-tabs-no-border">
+      <div class="govuk-tabs app-tabs-no-border" data-module="govuk-tabs">
```

Comment:
my plan for this (sorry, should have said/made it obvious) was to have them as real pages, that way when you change the organisation name you get redirected to the organisation details page/tab

Reply:
(also that is what I'm thinking for the donor/attorney/certificate provider dashboard when we get round to the pagination ticket)

Reply:
Makes sense with the name change use case. I'll update 👍